lib/identity_doc_auth/acuant/responses/get_results_response.rb

```diff
@@ -75,6 +83,29 @@ def raw_alerts
           parsed_response_body['Alerts'] || []
         end
 
+        def raw_regions
+          parsed_response_body['Regions'] || []
+        end
+
+        def regions_by_id
+          @regions_by_id ||= raw_regions.index_by { |region| region["Id"] }
+        end
+
+        def raw_images_data
+          parsed_response_body['Images'] || []
+        end
+
+        def processed_alerts
+          @processed_alerts ||= process_raw_alerts(raw_alerts)
+        end
+
+        def process_images_data
+          raw_images_data.index_by do |image|
+            image.except!('Uri')
```

Comment:
IMO if we're just mutating, I think just deleting the key would be clearer?

```suggestion
            image.delete('Uri')
```

Reply:
Yeah, I'm not certain why I went with except! probably just the first thing that came into my head.